carbonmark/pages/users/[user].tsx

```diff
@@ -70,7 +83,16 @@ const resolveDomain = async (params: { domain: string; locale?: string }) => {
  * Attempts to resolve a valid user handle. Throws if handle can't be resolved.
  * */
 const resolveHandle = async (params: { handle: string; locale?: string }) => {
-  const carbonmarkUser = await getUsersWalletorhandle(params.handle);
+  let carbonmarkUser: User | null = null;
+  try {
+    carbonmarkUser = await getUsersWalletorhandle(params.handle);
```

Comment:
Nit: might be a good chance to rename this?

```suggestion
    carbonmarkUser = await getUsersWalletOrHandle(params.handle);
```

Reply:
Unfortunately this one is generated from the API spec too. Will see what we can do.